fs.js

```diff
@@ -94,3 +102,14 @@ exports.makeDirSync = path => {
 		throw new CpFileError(`Cannot create directory \`${path}\`: ${err.message}`, err);
 	}
 };
+
+if (fs.copyFileSync) {
+	exports.copyFileSync = (src, dest, flags) => {
+		try {
+			fs.copyFileSync(src, dest, flags);
+		} catch (err) {
+			throw new CpFileError(`Cannot copy from \`${src}\` to \`${dest}\`: ${err.message}`, err);
+		}
+	};
+	exports.constants = fs.constants;
```

Comment:
Since we don't do any logic around this I'd rather `require` it directly from `fs` in `index.js`:

```js
const fsConstants = require('fs').constants;
```